packages/coordinator/src/services/translation/request/dosage.ts

```diff
@@ -194,33 +194,19 @@ function stringifyFrequencyAndPeriod(dosage: fhir.Dosage) {
   const periodMax = repeat?.periodMax
   const periodUnit = repeat?.periodUnit
 
-  if (!frequency && !frequencyMax) {
```

Comment:
After looking at the refactor I'm confident its like for like.

An approach to take in future when refacting is to ensure there are tests for func we are refactoring. If no tests then write some and see them pass then do the refactoring and you'll know if anything has been broken with the refactor